pygments_promql/__init__.py

```diff
@@ -157,7 +157,8 @@ class PromQLLexer(RegexLexer):
             (r"==|!=|>=|<=|<|>", Operator),
             (r"and|or|unless", Operator.Word),
             # Metrics
-            (r"[_a-zA-Z][_a-zA-Z0-9]+", Name.Variable),
+            (r"[_a-zA-Z][_a-zA-Z0-9]+(\$\{?[_a-zA-Z0-9]*\})?[_a-zA-Z0-9]*", Name.Variable),
```

Comment:
It could be better to implement this specific rules creating a subclass of `PromQLLexer`, something like:

```python
class PromQLGrafanaLexer(PromQLLexer):
    name = "PromQLGrafana"
    aliases = ["promqlgrafana"]
    filenames = ["*.promql"]

    tokens["root"] = {...}  # new rules for Grafana
...
```
and adding the class to:
```python
__all__ = ["PromQLLexer", "PromQLGrafanaLexer"]
```
that will allow to choose a specific lexer to users only wanting pure promql or Grafana one.